app/core/components/collector/__init__.py

```diff
@@ -434,55 +436,12 @@ def load_programs(self, program_provider: ProgramProvider, data: Iterable[dict])
         if bad_program_count:
             logging.error(f'Could not parse {bad_program_count} programs.')
 
-    @staticmethod
-    def available_resources(site: Site,
-                            night_idx: NightIndex) -> FrozenSet[Resource]:
+    def available_resources(self,
+                            site: Site,
+                            night_indices: npt.NDArray[NightIndex]) -> List[FrozenSet[Resource]]:
```

Comment:
This could be not too efficient if we want to look for a specific value right. O(n^2) search, I think we are cover now because we just need to check that a value is in there.